auditlog/models.py

```diff
@@ -179,8 +178,9 @@ class Action:
                               related_name='+', verbose_name=_("actor"))
     remote_addr = models.GenericIPAddressField(blank=True, null=True, verbose_name=_("remote address"))
     timestamp = models.DateTimeField(auto_now_add=True, verbose_name=_("timestamp"))
-    additional_data = JSONField(blank=True, null=True, verbose_name=_("additional data"))
-
+    additional_data_json = models.TextField(blank=True,
```

Comment:
I'm not sure if this is the right name for the column to be honest, since it's infact not a JSON field anymore but just plain text (since there is a real JSONField in this new Django era). 

How about just naming it like that (and removing the rename migration and updating the model property)?
```suggestion
    additional_data_text = models.TextField(blank=True,
```